src/main/java/com/microsoft/store/partnercenter/auditrecords/AuditRecordsCollection.java

```diff
@@ -47,13 +49,14 @@ public AuditRecordsCollection(IPartner rootPartnerOperations)
 		}
 
 		Collection<KeyValuePair<String, String>> parameters = new ArrayList<KeyValuePair<String, String>>();
+		DateFormat df = new SimpleDateFormat("YYYY-mm-dd");
 
 		parameters.add
 		(
 			new KeyValuePair<String, String>
 			(
 				PartnerService.getInstance().getConfiguration().getApis().get("GetAuditRecordsRequest").getParameters().get("StartDate"),
-				startDate.toString()
+				df.format(startDate) 
```

Comment:
@isaiahwilliams I believe that the Java Date format should have been "yyyy-MM-dd", since that is the right way to represent 2020-02-06 or any other date. The APIs should be more generic, they should expect `java.util.Date` as argument instead of creating interface dependency on `org.joda.time.DateTime`.